### DISCUSSION TITLE
Can I set client_max_body_size for proxy configuration?

### DISCUSSION THREAD

**Discussion by mureev** [nginx-proxy/nginx-proxy#1935] [Q&A] [answered]
Hello everyone,

My goal is to allow to upload big files to nexcloud behind acme-companion proxy. I would love to edit nginx config to add something like this "client_max_body_size 1G;".

Unfortunatlly with my basic usage the config is generated each time companion container starts.

What is the best way to pass such a config though acme to nginx vhost?

Thanks!
C

**Comment by buchdag** [comment, marked as answer]
Hi.

There are example of this on [proxy wide](https://github.com/nginx-proxy/nginx-proxy#proxy-wide) and [per-VIRTUAL_HOST](https://github.com/nginx-proxy/nginx-proxy#per-virtual_host) configuration documentation 👍

**Reply by NilsBaumgartner1994** [reply]
Would be nice to be able to set this with docker-compose env variable
